src/Orleans.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
```

Comment:
just for the record. I personally do it. And I asked another person from the core team too and he does it. I didn't do it to you personally. So don't take it personal. 

avoid unnecessary change is the first thing I learnt in my job. If I did this, any team I worked before for would be against. 

And it is not hard to revert this at all.